src/Games/NexusMods.Games.StardewValley/Diagnostics.cs

```diff
@@ -134,4 +134,81 @@ internal static partial class Diagnostics
             .AddValue<string>("ModVersion")
         )
         .Finish();
+
+    [DiagnosticTemplate]
+    [UsedImplicitly]
+    internal static IDiagnosticTemplate GameVersionOlderThanMinimumGameVersionTemplate = DiagnosticTemplateBuilder
+        .Start()
+        .WithId(new DiagnosticId(Source, number: 8))
+        .WithTitle("Game Version older than supported by SMAPI")
+        .WithSeverity(DiagnosticSeverity.Critical)
+        .WithSummary("The minimum supported game version of SMAPI {SMAPIVersion} is {MinimumGameVersion}")
+        .WithDetails("""
+SMAPI version {SMAPIVersion} requires the game version to be at least {MinimumGameVersion}.
+The current game version is {CurrentGameVersion}.
+
+Due to this version mismatch, the game will **crash** on startup.
+You can solve this issue by either updating your game or downgrading SMAPI.
+
+The last supported SMAPI version for game version {CurrentGameVersion} is {LastSupportedSMAPIVersionForCurrentGameVersion}.
```

Comment:
Use same language as the other message:
`The newest supported SMAPI version for game version...`
It is clearer to the user

Also use 
`... for game version {CurrentGameVersion} is SMAPI v{LastSupportedSMAPIVersionForCurrentGameVersion}`
-> `... for game version 1.5.6 is SMAPI v3.18.6`

Since the message is showing two version next to each other this helps distinguish them